### DISCUSSION TITLE
auth()->user() at mount function

### DISCUSSION THREAD

**Discussion by kalwinskidawid** [livewire/livewire#5317] [1. Help]
Hey, 
I'm quite new with livewire but looks quite similar to vue, but I have a problem with mount.
I'm trying bind model User, If I dump element then I'll get correct User model, however when I'm checking values in devTools then I got empty array. Where is a problem?
EDIT. I'm work with jetstream+livewire

```
public $state = [];
public User $user;
    public function mount() {
      
            $this->user = auth()->user();
            dd($this->user);//here I'll get object
    }
```
![image](https://user-images.githubusercontent.com/9535316/197299226-63fe62ec-4547-4038-8868-dae50bd4cd1d.png)

**Comment by faramarzJafari** [comment]
Did you use factory for make your model in your test ?

**Reply by kalwinskidawid** [reply]
Yeah

**Comment by faramarzJafari** [comment]
So you use create() method  or make () methode when you want to use
facrory  ?

**Reply by kalwinskidawid** [reply]
I use create() in factory

**Reply by kalwinskidawid** [reply]
In registration controller also use User::create()

**Comment by faramarzJafari** [comment]
Try in your test
User::factory()->create();

**Comment by joshhanley** [comment]
@kalwinskidawid I wouldn't trust the devtools as it doesn't show everything.

For security purposes, if any Eloquent model or Eloquent collection is stored in a public property, by default none of the models attributes will be sent to the front end. To do this you need to add the attributes to the `$rules` array. See here in the docs for details https://laravel-livewire.com/docs/2.x/properties#binding-models

Hope this helps!
